spec/features/event/ranking_league_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
```

Comment:
Please rename this file to `ranking_event_spec.rb` and add negative tests for Tournaments and Rankinglists regarding the generation of Rankings